### PR TITLE
Jett's new auto

### PR DESCRIPTION
**Pull request by JettK**
Sam here it is

### CODE REVIEW

File: src/main/java/org/usfirst/frc/team2928/Robot.java [hunk 1]
```diff
@@ -58,13 +59,13 @@ public void robotInit() {
...
         autoSelector.addObject("Drive 5 feet forward [Test]", Auto.TEST_DISTANCE);
         SmartDashboard.putData("Auto Chooser", autoSelector);
-
         startingPositionSelector = new SendableChooser<>();
         startingPositionSelector.addObject("Left", Field.FieldPosition.LEFT);
         startingPositionSelector.addDefault("Middle", Field.FieldPosition.MIDDLE);
         startingPositionSelector.addObject("Right", Field.FieldPosition.RIGHT);
         SmartDashboard.putData("Starting Position", startingPositionSelector);
-        CameraServer.getInstance().startAutomaticCapture();
+        CameraServer.getInstance().startAutomaticCapture(0);
```
Comment (alice157) [inline src/main/java/org/usfirst/frc/team2928/Robot.java:67]: Will we have 2 cameras?

**Review by alice157 — APPROVED** [review]
If we're using 2 cameras, feel free to merge. In the future, please avoid uploading files on Github, use git or IntelliJ's git client to commit and push.

File: src/main/java/org/usfirst/frc/team2928/Robot.java (hunk 1 above)
Comment (JettK) [reply src/main/java/org/usfirst/frc/team2928/Robot.java:67]: Yep, we got two cameras now and they all work

Comment (JettK) [reply src/main/java/org/usfirst/frc/team2928/Robot.java:67]: I'll use intellij git next time, this is my first time really using github

**Pull request merged**